tests/sentry/mail/test_actions.py

```diff
@@ -197,7 +197,7 @@ def test_hack_mail_workflow(self):
             )
 
         assert len(mail.outbox) == 3
-        sent_out_to = [x for out in mail.outbox for x in out.to]
-        assert sent_out_to == [self.user.email, gil_workflow.email, dan_workflow.email]
+        sent_out_to = {x for out in mail.outbox for x in out.to}
+        assert sent_out_to == {self.user.email, gil_workflow.email, dan_workflow.email}
```

Comment:
Rather than using sets, could we sort the lists here? Otherwise we'd miss duplicate emails

Reply:
done